### PR TITLE
Aria hidden

### PR DESCRIPTION
**Pull request by jquense**
related to #12 

This is a first pass at doing _some_ aria-hidden stuff. All of this is pretty ugly, from a component perspective. Unfortunately doing this is really necessary for a modal to appear like a modal to a screen reader, Without the aria-hidden on the rest of the page content the reader will shift focus (without regard for our focus trapping code) to the rest of the page.

This is rendered very useless though, if the `container` isn't a parent of the application, where modals would be its siblings.

### CODE REVIEW

File: src/utils/manageAriaHidden.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  [].forEach.call(container.children, node => {
+    if (mount.indexOf(node) === -1 && isHidable(node)){
+      cb(node);
+    }
+  });
+};
+
+export function ariaHidden(show, node){
+  if (!node) {
```
Comment (AlexKVal) [inline src/utils/manageAriaHidden.js]:
```js
if (!node || !isHidable(node)) {
```
?

**Comment by taion** [comment]
That caveat regarding body is probably fine, right? If your container isn't the body, then even the visible backdrop doesn't get rendered over everything - just over the container.

**Comment by jquense** [comment]
yes but that also assumes that the modal will be confined to the container with some css. the case I am sort of thinking of is having like a div off the body where everything is appended... We can't really solve all the issues I guess, we can at least suggest in the docs that folks just use the document body as the container.

**Comment by taion** [comment]
(I don't understand why you'd do that. It seems like extra work for no good reason. I don't understand contained modals at all anyway, though.)

**Comment by jquense** [comment]
why do we event allow that for modals? We should reexamine even showing people that in RB i think...

**Pull request merged**